test/i6pn_clustered_test.py

```diff
@@ -39,17 +39,6 @@ def test_experimental_group(servicer, client):
         assert fn3.i6pn_enabled is True
 
 
-def test_spawn_experimental_group(client, servicer, monkeypatch):
-    # We need to set a custom function body here since grouped function's kwargs
-    # aren't compatible with the default servicer function body.
-    @servicer.function_body
-    def grouped_func(*args, **kwargs):
-        return sum(args)
-
+def test_spawn_experimental_cluster(client, servicer, monkeypatch):
```

Comment:
Nit: but why "spawn" in the name if we are exercising `.remote`? (I know you're just modifying  an existing test here).

Reply:
Good point 😅 I'll rename it to `test_run_experimental_cluster`